### ISSUE TITLE
Improve getInfo API method

### ISSUE THREAD

**Issue by sangaman** [ExchangeUnion/xud#120]
We have a `getInfo` method that is currently very basic - it simply calls the underlying lnd `getInfo` method and passes this along. We should expand this call to provide more meaningful data about the state of an `xud` instance. Some ideas:

- Number of peers connected
- Number of trading pairs/currencies supported
- Number of peer orders and own orders.
- Info about the Raiden node.

**Comment by michael1011** [comment]
I wouldn't show the whole `getInfo` of LND because most of the information is pretty useless to end users. I think this would be enough:

- number of active and inactive channels
- number of pending channels (if there are any)
- uri of the node
- maybe the block height to make sure that the node is fully synced

And for Raiden as they don't show more information:

- account address for the Raiden node
- number of channels

**Comment by itsbalamurali** [comment]
I would also suggest node / xud version along with order book protocol version just in case.

**Comment by michael1011** [comment]
Good idea. Getting the version of LND is possible with the API. But what about Raiden? Should their version just be hard coded until they add the version to the API?

**Comment by sangaman** [comment]
All good ideas, I agree that we don't need the entire `getinfo` output from `lnd`. I just threw out a partial list of ideas. The `RaidenClient` class already has some methods built in that return info about the state of the node that can be used.
